app/controllers/administrateurs/administrateur_controller.rb

```diff
@@ -1,12 +1,12 @@
 module Administrateurs
   class AdministrateurController < ApplicationController
     before_action :authenticate_administrateur!
+    helper_method :is_administrateur_through_procedure_administration_as_manager?
```

Comment:
```suggestion
    helper_method :administrateur_through_procedure_administration_as_manager?
```

En `ruby` comme nous avons droit aux `?` pas besoin de `is_` je pense :)

Reply:
Même remarque que pour l'instructeur `administrateur_as_manager?`